test/unified_format.py

```diff
@@ -1699,6 +1719,15 @@ def run_operations(self, spec):
             else:
                 self.run_entity_operation(op)
 
+    def run_operations_and_throw(self, spec):
+        for op in spec:
+            if op["object"] == "testRunner":
+                self.run_special_operation(op)
+            else:
+                result = self.run_entity_operation(op)
+                if isinstance(result, Exception):
+                    raise result
```

Comment:
Why re-raise this exception after ignoreResultAndError or expectError? Does the spec say to do that somewhere?

Reply:
Yes, "Ensure that neither ignoreResultAndError nor expectError interfere with propagating exceptions from a callback operation to withTransaction"

Reply:
Where is that written? That line doesn't appear in the spec repo.

Reply:
That is from the ticket description, there is more discussion on the original spec PR.